ff -rup PokerTH-0.9.3-src.orig/src/net/common/ircthread.cpp PokerTH-0.9.3-src/src/net/common/ircthread.cpp
--- PokerTH-0.9.3-src.orig/src/net/common/ircthread.cpp	2012-02-11 17:57:34.000000000 +0100
+++ PokerTH-0.9.3-src/src/net/common/ircthread.cpp	2012-02-13 21:45:06.178062234 +0100
@@ -19,7 +19,8 @@
 #include <net/socket_helper.h>
 #include <net/ircthread.h>
 #include <net/socket_msg.h>
-#include <libircclient/libircclient.h>
+#include <libircclient.h>
+#include <libirc_rfcnumeric.h>
 #include <boost/algorithm/string/predicate.hpp>
 #include <queue>
 #include <sstream>
